src/pages/workspaces/workspace/analysis/AzureComputeModal.js

```diff
@@ -24,7 +24,7 @@ import * as Utils from 'src/libs/utils'
 const titleId = 'azure-compute-modal-title'
 
 export const AzureComputeModalBase = ({
-  onDismiss, onSuccess, workspace: { workspace: { namespace, name: workspaceName, workspaceId } }, runtimes, hideCloseButton = false
+  onDismiss, onSuccess, onError = onDismiss, workspace: { workspace: { namespace, name: workspaceName, workspaceId } }, runtimes, hideCloseButton = false
```

Comment:
Why not just pass `onDismiss` to `withErrorReportingInModal`? `onDismiss` is both the default value _and_ the value that seems to be passed in explicitly in all cases (in the other file in this PR).

Reply:
It does diverge in a few cases, I'd like the caller to have to think about it, but protect against them forgetting to pass it in.

Reply:
Ahh, okay. I didn’t notice the higher-order component from `withModalDrawer` so I didn’t trace the props back far enough. The varied `onError` comes from the use of `AzureComputeModal` in `Analyses.js`.